website/_includes/head_custom.html

```diff
@@ -1,18 +1,19 @@
-<!-- Generated using https://realfavicongenerator.net. -->
+<!-- Generated from https://realfavicongenerator.net/ -->
 <link rel="apple-touch-icon" sizes="180x180" href="/assets/images/apple-touch-icon.png">
 <link rel="icon" type="image/png" sizes="32x32" href="/assets/images/favicon-32x32.png">
 <link rel="icon" type="image/png" sizes="16x16" href="/assets/images/favicon-16x16.png">
 <link rel="manifest" href="/assets/images/site.webmanifest">
 <link rel="mask-icon" href="/assets/images/safari-pinned-tab.svg" color="#5bbad5">
-<link rel="shortcut icon" href="/favicon.ico">
+<link rel="shortcut icon" href="/assets/images/favicon.ico">
 <meta name="msapplication-TileColor" content="#da532c">
 <meta name="msapplication-config" content="/assets/images/browserconfig.xml">
 <meta name="theme-color" content="#ffffff">
 
-<!-- XXX: The theme does not natively support both light and dark mode. Drop
-this section once https://github.com/just-the-docs/just-the-docs/issues/234 is
-resolved. -->
+<!-- Support light and dark mode, as it's not natively supported.
+    See: https://github.com/just-the-docs/just-the-docs/issues/234 -->
 <link rel="stylesheet" href="{{ '/assets/css/just-the-docs-eps-light.css' | relative_url }}"
-    media="(prefers-color-scheme: light)">
+      media="(prefers-color-scheme: light)">
```

Comment:
These spaces seem a bit weird 👀. Is that because of a formatter?

Reply:
Seems to have changed during cherry-pick from @oxkitsune's [branch](https://github.com/PicnicSupermarket/error-prone-support/blob/525df33e420f29c14f86d40ee30ea28229d8f712/website/_includes/head_custom.html) to [mine](https://github.com/PicnicSupermarket/error-prone-support/commit/601b4acaedf12bd7749cb453d6f9909209e32d78#diff-217919e1351e96e8a2cd2959c63c9d31f19cde00277454f1c470f2ec545de81a). This formatting happens to me as part of IntelliJ formatting (CTRL-ALT-L). Currently no CI formatting/linting is in place for these files, nor could I find any so quickly to run it as a one-off. I'm happy with reverting if you'd like.

Reply:
Yeah let's do IntelliJ formatting for now :).